utils/telegram.py

```diff
@@ -62,11 +62,11 @@ def broadcast(self, message, filter_str='', sticker=None):
                 conversations.append(conversation)
         self.logger.log(f'broadcasting message in conversations {conversations}')
 
-        for chunk in self.chunks(message):
-            for conversation in conversations:
+        for conversation in conversations:
+            for chunk in self.chunks(message):
```

Comment:
just FYI, you create `n` string separations over here where n is the amount of conversations